config_db.go

```diff
@@ -53,11 +53,10 @@ var (
 	hostinitKey                     = bsonutil.MustHaveTag(ServiceFlags{}, "HostinitDisabled")
 	monitorKey                      = bsonutil.MustHaveTag(ServiceFlags{}, "MonitorDisabled")
 	alertsKey                       = bsonutil.MustHaveTag(ServiceFlags{}, "AlertsDisabled")
-	taskrunnerKey                   = bsonutil.MustHaveTag(ServiceFlags{}, "TaskrunnerDisabled")
```

Comment:
Also change in testdata/smoke_config.yml.

Reply:
Is it not changed?

Reply:
Sorry, totally missed it.